Library/Formula/twoping.rb

```diff
@@ -0,0 +1,18 @@
+class Twoping < Formula
+  desc "Ping utility to determine directional packet loss"
+  homepage "http://www.finnie.org/software/2ping/"
+  url "http://www.finnie.org/software/2ping/2ping-3.0.1.tar.gz"
+  sha256 "d6997cd1680151e6f7d5e60137d45cd41bf385d26029878afdaaf5dc4f63dcc4"
+
+  depends_on "Python"
```

Comment:
Actually, 2ping is an easy-install program which relies on the installed twoping module (not a single script).  With the Python formula removed, it warns about site-packages during install:

```
ryan@ndnd:~$ brew install 2ping
==> Downloading http://www.finnie.org/software/2ping/2ping-3.0.1.tar.gz
Already downloaded: /Library/Caches/Homebrew/twoping-3.0.1.tar.gz
==> python -c import setuptools... --no-user-cfg install --prefix=/usr/local/Cellar/twoping/3.0.1 --single-version-externally-managed --record=installed.txt
==> Caveats
Python modules have been installed and Homebrew's site-packages is not
in your Python sys.path, so you will not be able to import the modules
this formula installed. If you plan to develop with these modules,
please run:
  mkdir -p /Users/ryan/Library/Python/2.7/lib/python/site-packages
  echo 'import site; site.addsitedir("/usr/local/lib/python2.7/site-packages")' >> /Users/ryan/Library/Python/2.7/lib/python/site-packages/homebrew.pth
==> Summary
🍺  /usr/local/Cellar/twoping/3.0.1: 27 files, 244K, built in 2 seconds
```

and 2ping fails:

```
ryan@ndnd:~$ 2ping
Traceback (most recent call last):
  File "/usr/local/bin/2ping", line 5, in <module>
    from pkg_resources import load_entry_point
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 2793, in <module>
    working_set.require(__requires__)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 673, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 576, in resolve
    raise DistributionNotFound(req)
pkg_resources.DistributionNotFound: 2ping==3.0.1
```

The stated workaround works but users can't be expected to do that, correct?  Does that mean homebrew-managed python is effectively required?